source/find_getmore_killcursors_commands.rst

```diff
@@ -263,17 +263,39 @@ The driver **SHOULD NOT attempt to emulate the behavior seen in 3.0 or earlier**
 Tailable cursors
 ^^^^^^^^^^^^^^^^
 
-Tailable cursors have some fundamental changes compared to the existing **OP_QUERY** implementation. To create a tailable cursor you execute the following command:
+By default most cursors are non-tailable, for example, a ``find`` that exhausts when all results for the filter have been satisfied.
+MongoDB also supports creating cursors that "tail" or follow the target namespace for new data.
+Querying capped collections and change streams are some examples of tailable cursor use cases.
+A tailable cursor can receive ``getMore`` responses with an empty ``nextBatch`` array, this does not indicate that the cursor has been exhausted.
 
-.. code:: javascript
-
-    var b = db.runCommand({ find:"t", tailable: true });
+In addition to considering a find cursor tailable, an ``awaitData`` flag MAY be sent on the initial command.
+This will request that the server wait to respond to the client's ``getMore`` command until new data is available.
+The amount of time the server will wait for new data is based on the ``maxTimeMS`` field of the ``getMore`` (or server default if unspecified).
+If the time does expire an empty batch will be returned.
+A ``maxTimeMS`` field cannot be sent if the cursor was not configured with ``awaitData=true``.
 
-To create a tailable cursor with **tailable** and **awaitData**, execute the following command:
+To create a tailable ``find`` cursor you execute the following command:
 
-.. code:: javascript
+.. _find: https://www.mongodb.com/docs/manual/reference/command/find/
 
-    var b = db.runCommand({ find:"t", tailable: true, awaitData: true });
+.. code:: typescript
+
+  interface FindCommand {
+    /** The namespace to run the find command on */
+    find: string;
+    /** The filter to control what documents are returned */
+    filter: BSONDocument;
+    /** Informs the server whether to keep the cursor open even when there are no results to satisfy the query */
+    tailable?: boolean;
+    /**
+     * Informs the server whether to block on a `getMore`'s `maxTimeMS` before returning an empty `nextBatch`.
+     * This must be set if getMores are to include `maxTimeMS` values.
+     */
+    awaitData?: boolean;
+    /** Controls the amount of milliseconds the server will allow the operations to run for */
+    maxTimeMS?: PositiveIntegerNumber;
+    // ... Non-normative: additional fields omitted
```

Comment:
I got the phrase from web specs: https://developer.mozilla.org/en-US/docs/Glossary/Non-normative Basically, an example but not a requirement, which I think applies to the interfaces b/c there's many variations depending on language

But I'll remove it since we don't use it elsewhere nor have it in our glossaries

Reply:
Thanks for the link. TIL.